src/main/java/fi/metatavu/dcfb/server/rest/ItemsApiImpl.java

```diff
@@ -298,7 +309,8 @@ public Response listItems(String categoryIdsParam, String locationIdsParam, Stri
     }
 
     SearchResult<fi.metatavu.dcfb.server.persistence.model.Item> searchResult = itemController.searchItems(nearLat, nearLon, 
-        categories, locations, search, getLoggerUserId(), includeExhausted != null ? includeExhausted.booleanValue() : false, firstResult, maxResults, sorts);
+      userIds, categories, locations, search, getLoggerUserId(), includeExhausted != null ? includeExhausted.booleanValue() : false, 
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove the literal "false" boolean value. ([squid:S1125](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1125))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.dcfb-api%3Adcfb-api&pullRequest=32&issues=AWXSgwp4lgbYnohYHn6C&open=AWXSgwp4lgbYnohYHn6C)

[ISSUE_KEY]: AWXSgwp4lgbYnohYHn6C